src/main/java/htsjdk/samtools/SAMFileHeader.java

```diff
@@ -325,10 +324,11 @@ public void setAttribute(final String key, final Object value) {
      */
     @Override
     public void setAttribute(final String key, final String value) {
-        super.setAttribute(key, value);
-        if (key.equals(SORT_ORDER_TAG)) this.setSortOrder(SortOrder.valueOf(value));
-        else if (key.equals(GROUP_ORDER_TAG)) this.setGroupOrder(GroupOrder.valueOf(value));
-        else super.setAttribute(key, value);
+        if (key.equals(SORT_ORDER_TAG)) {
+            this.setSortOrder(SortOrder.valueOf(value));
+        } else if (key.equals(GROUP_ORDER_TAG)) {
+            this.setGroupOrder(GroupOrder.valueOf(value));
+        } else super.setAttribute(key, value);
```

Comment:
lol thanks @nh13 
I think it's 👍 but someone who actually understands the code should probably look at it too.